functions/index.js

```diff
@@ -25,3 +29,25 @@ exports.databaseBackup = functions.pubsub.schedule('every 24 hours').onRun((cont
       throw new Error('Export operation failed');
     });
 });
+
+exports.addUser = functions.auth.user().onCreate((user) => {
+  const defaultPicture = "https://t4.ftcdn.net/jpg/00/64/67/63/240_F_64676383_LdbmhiNM6Ypzb3FM4PPuFP9rHe7ri8Ju.jpg";
+
+  const random = Math.ceil(Math.random() * 20000);
+  const id = user.uid;
+  const name = user.displayName ? user.displayName : `User ${random}`;
+  const userPhoto = user.photoURL ? user.photoURL: defaultPicture;
+  const email = user.email ? user.email : `${random}@google.com`;
```

Comment:
and for this one we don't need the email check. They ll always have email. we didn't have that for hackathon.
Just have `const email = user.email`